files/en-us/web/api/node/comparedocumentposition/index.html

```diff
@@ -113,5 +126,5 @@ <h2 id="Browser_compatibility">Browser compatibility</h2>
 <h2 id="See_also">See also</h2>
 
 <ul>
-  <li><code><a href="/en-US/docs/DOM/Node.contains">Node.contains()</a></code></li>
+  <li>{{DOMxRef("Node.contains()"}}</li>
```

Comment:
I think the build fails because this is not a valid macro call.

```suggestion
  <li>{{DOMxRef("Node.contains()")}}</li>
```

Reply:
Committing this so we can see the results of the tests and swiftly lands this once the last comment is fixed.